packages/app/src/specs/SpecsList.vue

```diff
@@ -307,6 +309,10 @@ const emit = defineEmits<{
   (e: 'showCreateSpecModal'): void
 }>()
 
+const cloudProjectId = computed(() => {
+  return props.gql?.currentProject?.config?.find((item: { field: string }) => item.field === 'projectId')?.value
```

Comment:
Could we just add the `projectId` field to the fields being queried under `currentProject` and reference that?

Reply:
let me double check, I have a terrible feeling that that project ID is a different thing than this

Reply:
Sweet, yeah this is an improvement, I'll make the change. We've definitely overburdened the words "project" and "id". I feel like there is a 3rd thing that's not in this screenshot that represents the unique ID of a local project, as in a "directory we've opened in Cypress" not a project in the cloud sense.

![Screen Shot 2022-08-12 at 2 44 50 PM](https://user-images.githubusercontent.com/8340719/184423697-ee34aecb-f720-4f5c-824e-0bc36248cc7a.png)